File: Program94.c
/*   
iRow  = 4 
iColm = 4
     j

   1  *  3  *     

i  1  *  3  *     

   1  *  3  *      

  

*/

#include<stdio.h>
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//              Function Name       : Pattern
//              Description         : Function to print Pttern 
//              INput Argument      : Integer ,Integer
//              Output Argument     : Integer ,Integer
//              Date                : 31 October 2023
//              Author              : Swati Vinayak Lahane
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Function to display Star in even column and printf numeric value in odd Row number using  4 by 4 matrix

void Display(int iRow ,int iColm)
{
    int iCnt1 = 0;
    int iCnt2 = 0;
   
    //    1           2             3
    for(iCnt1 = 1; iCnt1 <= iRow; iCnt1++)// inner loop
    {   
        for(iCnt2 = 1; iCnt2 <= iColm; iCnt2++)//Outer loop
        {
            if(iCnt2 % 2 != 0)
            {
                 printf("%d\t",iCnt2);  //4
            }
           else
           {
               printf("*\t");
           }
        }
        printf("\n\n");
    }

}
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//
//                      Entry point function from where execution gets starts
//
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
int main()
{ 
   int iNo1 =0 ,iNo2 = 0;

   printf("Enter the number of rows:\n");
   scanf("%d",&iNo1);

   printf("Enter the number of colms:\n");
   scanf("%d",&iNo2);

   Display(iNo1,iNo2); //Function call

 return 0;
}

File: Java Programs/Problems on Pattern/Program94.java
/*   
iRow  = 4 
iColm = 4
     j

   $  &  $  &     

i  $  &  $  &     

   $  &  $  &      

   $  &  $  &    

*/

import java.util.*;
 
class Arithematic 
{
     public void Dispaly(int iNo1, int iNo2)
     {
         int iCnt1 = 0;
         int iCnt2 = 0;
         
         for(iCnt1 = 1; iCnt1 <= iNo1; iCnt1++)
         {
             for(iCnt2 = 1; iCnt2 <= iNo2; iCnt2++) 
             {
                 if(iCnt2 % 2 == 0)
                 {
                     System.out.print("*"+"\t");
                 } 
                 else
                 {
                     System.out.print(iCnt2 +"\t");
                 }
                 
             }
             System.out.println();
         }
     }
}
class Program23
{   
    public static void main(String arg[])
    {

    Scanner sobj = new Scanner(System.in);
    Arithematic aobj = new Arithematic();

    int iValue1 = 0;
    int iValue2 = 0;

    System.out.println("Enter Number of Rows :");
    iValue1 = sobj.nextInt();

    System.out.println("Enter number of colms :");
    iValue2 = sobj.nextInt();

    aobj.Dispaly(iValue1,iValue2);

}

}